README.md

```diff
@@ -184,47 +231,23 @@ rootController.add(childController: controller, to: containerView)
 rootController.remove(childController: controller)
 ```
 
-### Keychain
-
-We have a cleaner way to use the `Keychain`. Define the user defaults by extending the `Keys` class.
-
-```swift
-extension Keys {
-  // Writes a string object to the keychain with the 'stringValue' key.
-  static let stringValue = Key<String?>("stringValue")
-}
-```
-
-You can read/write the from/to the `Keychain` by using the `subscript` on the `Keychain` class.
-
-```swift
-Keychain[.stringValue] = "A string value"
-print(Keychain[.stringValue]) // Prints 'A string value'
-```
-
-In some cases you want to be able to set additional keychain query paramaters on an item.
-
-```swift
-static let noBackupValue = Key<String?>("noBackup", {
-  return [kSecAttrAccessible as String: kSecAttrAccessibleAlwaysThisDeviceOnly]
-}())
-```
-
-### Localization
+## Contrinute
 
-Localize a key in no time with this handy localization function.
+### How to contribute ❓
 
-```swift
-let key = "this_is_your_localization_key"
-print(key.localizedString)
-// The debug console will print the localized
-// string found in your .strings file.
-```
+1. Add a Github issue describing the missing functionality or bug.
+2. Implement the changes according to the `Swiftlint` coding guidelines.
+3. Fully test the added changes.
+4. Send a pull-request.
 
-## Author
+### Contributors 🤙
 
-Jelle Vandebeeck, jelle.vandebeeck@icapps.com
-Hannes Van den Berghe, hannes.vandenberghe@icapps.com
+- Jelle Vandebeeck, [@fousa](https://github.com/fousa)
+- Dylan Gyesbreghs, [@dgyesbreghs](https://github.com/dgyesbreghs)
+- Hans Van Herreweghe, [@herre](https://github.com/herre)
```

Comment:
ik denk niet dat ik hier al heb toe bijgedragen 🤷‍♂️

Reply:
Ik ben in de commits gaan kijken en je stond er bij ;)